mne/viz/ica.py

```diff
@@ -272,8 +272,10 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         The ICA solution.
     inst : instance of Epochs or Raw
         The data to use in plotting properties.
-    %(picks_base)s the first five sources.
-        If more than one components were chosen in the picks,
+    picks : str | list | slice | None
+        Components to include. Slices and lists of integers will be interpreted
+        as component indices. None (default) will use the first five
+        components. If more than one components were chosen in the picks,
         each one will be plotted in a separate figure.
```

Comment:
```suggestion
```suggestion
        Components to include. Slices and lists of integers will be interpreted
        as component indices. ``None`` (default) will use the first five
        components. Each component will be plotted in a separate figure.
```

Reply:
Looks good to me!